src/components/GoodsList.tsx

```diff
@@ -0,0 +1,20 @@
+type Props = {
+  goods: string[];
+};
+
+export const GoodsList: React.FC<Props> = ({ goods }) => (
+  <ul>
+    <ul>
+      {goods.map(good => {
+        return (
+          <li
+            data-cy="Good"
+            key={good}
+          >
+            {good}
+          </li>
+        );
+      })}
+    </ul>
+  </ul>
```

Comment:
Why did you wrap it to `<ul>` element 2 times ?

Reply:
It was wrapped with two ul's in the original App component